packages/strapi-parts/src/Accordion/Accordion.stories.mdx

```diff
@@ -19,7 +20,17 @@ This is the doc of the `Accordion` component
 
 ## Accordion
 
-Description...
+Accordions are collapsable blocks that vertically hide a stack of content. Clicking on the block reveals the associated content. 
+
+**Best practices**
+
+- Use accordion when you need big and/or related chunks of content to be organized in a same place.
+- Use accordion for lower priority information.
+- Use to shorten pages and increase readability.
+- The chevron icon pointing down indicates a collapsed state and pointing up indicates an expanded state.
+- Hover and active states should be used anytime it's possible.
+- Content should not scroll within an accordion.
```

Comment:
We don't allow this in the accordion. People can however tweak the content themselve (using an itermediate HTML element with a fixed height and an overflow). But by default, no scrollbar. Should we mention this line?

Reply:
No because we don't want users to add a scrollbar, hence this line :D